presto-delta/src/main/java/com/facebook/presto/delta/DeltaSessionProperties.java

```diff
@@ -14,37 +14,26 @@
 package com.facebook.presto.delta;
 
 import com.facebook.presto.cache.CacheConfig;
-import com.facebook.presto.hive.HiveClientConfig;
 import com.facebook.presto.spi.ConnectorSession;
-import com.facebook.presto.spi.schedule.NodeSelectionStrategy;
 import com.facebook.presto.spi.session.PropertyMetadata;
 import com.google.common.collect.ImmutableList;
-import io.airlift.units.DataSize;
 
 import javax.inject.Inject;
 
 import java.util.List;
 
-import static com.facebook.presto.common.type.VarcharType.VARCHAR;
-import static com.facebook.presto.hive.HiveSessionProperties.dataSizeSessionProperty;
 import static com.facebook.presto.spi.session.PropertyMetadata.booleanProperty;
 
 public final class DeltaSessionProperties
 {
     private static final String CACHE_ENABLED = "cache_enabled";
```

Comment:
Since this session property is not introduced in HiveCommonSessionProperties. We need to keep this to get added to different connectors.
And I didn't add this to HiveCommonSessionProperties since this is part of  `presto-cache` module which will create cycle dependency if we move it to `presto-hive-common`

Reply:
I just saw that this property is used in `HiveCachingHdfsConfiguration`
Let's add this comment in Delta and Iceberg connector similar to how it is added in Hudi Connector to remove confusion `// required by presto-hive module, might be removed in future`